build/source/engine/ascii_util.f90

```diff
@@ -76,10 +76,10 @@ subroutine split_line(inline,words,err,message)
  integer(i4b),intent(out)             :: err      ! error code
  character(*),intent(out)             :: message  ! error message
  ! declare local variables
- integer(i4b),parameter  :: cLen=2048
+ integer(i4b),parameter  :: cLen=8192
  character(len=cLen)     :: temp                  ! temporary line of characters
  integer(i4b)            :: iword                 ! loop through words
- integer(i4b),parameter  :: maxWords=100          ! maximum number of words in a line
+ integer(i4b),parameter  :: maxWords=1000         ! maximum number of words in a line
```

Comment:
issue #118 - both of these are good examples why centralizing this will be useful